### PR TITLE
Mondo 2.1: Antiforgery CORS HttpSysServer ResponseCaching Routing and Security

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

**Comment by ryanbrandenburg** [comment]
Failures were Kestrel and Hosting, will merge once I have approval.

**Review by natemcmaster — APPROVED** [review]

**Pull request merged**
